unstructured/partition/text_type.py

```diff
@@ -101,7 +109,13 @@ def is_possible_title(
         The minimum number of alpha characters the text needs to be considered a title
     language
         The two letter language code for the text. defaults to "en" for English
+    language_checks
+        If True, conducts checks that are language to the specified language.
```

Comment:
```suggestion
        If True, conducts checks that are specific to the chosen language.
```
Same as above, maybe indicate that this could be a performance hit?